ui/pages/Search/components/filters/LocusListSelector.jsx

```diff
@@ -12,14 +12,16 @@ class BaseLocusListDropdown extends React.Component
   render() {
     const { locusList, projectLocusListOptions, onChange } = this.props
     return (
-      <Dropdown
-        inline
-        selection
-        label="Gene List"
-        value={locusList.locusListGuid}
-        onChange={locusListGuid => onChange({ locusListGuid })}
-        options={projectLocusListOptions}
-      />
+      <div>
+        <Dropdown
+          inline
+          selection
+          label="Gene List"
+          value={locusList.locusListGuid === '' ? null : locusList.locusListGuid}
```

Comment:
having an equivalence check for an empty string and then assigning it to null is VERY risky and confusing to read. Why not just assign it to null if you want it to be null? Also, whats wrong with it just being undefined?